react-app/src/components/GamePage.js

```diff
@@ -360,22 +370,20 @@ class Game extends Component {
     let y = e.data.global.y;
 
     this.setState(() => ({
+      ...this.state,
       bins: this.state.bins.map((bin) => ({
         ...bin,
         hover: this.isSpriteInBin(x, y, bin.x, bin.y),
         shakeLife: 0,
         offsetX: 0,
       })),
-      trashList: this.state.trashList.map((item, i) =>
-        i !== this.selectedIndex
-          ? item
-          : {
-              ...item,
-              velocity: { x: 0, y: 0, rotation: 0 },
-              fixed: true,
-              x: x - this.dragStartScreenX + this.dragStartObjectX,
-              y: y - this.dragStartScreenY + this.dragStartObjectY,
-            }
+      selectedTrash: this.state.selectedTrash.map((item) => ({
```

Comment:
This logic is much easier to read without those pesky !=='s. FABULOUS.

Reply:
: )